lang/cs/Org.Apache.REEF.Bridge/Org.Apache.REEF.Bridge.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>Org.Apache.REEF.Bridge</id>
+    <version>$version$</version>
+    <title>Org.Apache.REEF.Bridge</title>
+    <authors>The Apache REEF project</authors>
+    <owners>The Apache REEF project</owners>
+    <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
+    <projectUrl>http://reef.incubator.apache.org/</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>REEF is a platform to run applications on distributed environment</description>
```

Comment:
We should have a description here, e.g. "The C# components of the bridge connecting the Java and C# side of the driver."